### DISCUSSION TITLE
change inside ( does not work

### DISCUSSION THREAD

**Discussion by JangoCG** [JetBrains/ideavim#446] [Q&A]
Hello everyone,
I just started using this plugin and I cant get the 

`change inside ( `

command to work:
Is this is a known bug? 
` change inside " `and 

`change inside ' `

works.

Steps to reproduce:

Go to a line where we have a () 
type ci(

This should result in the caret jumping into the parenthesis and deleting the content

**Comment by AlexPl292** [comment]
Hi, the `ci(` command should work out of the box, but the caret should be placed inside of the parenthesis already. It doesn't jump to the nearest parenthesis, however, as I remember, there is some vim plugin that performs the jump.
